components/Results/results.tsx

```diff
@@ -0,0 +1,150 @@
+/* eslint-disable @next/next/no-img-element */
+import { useState } from "react"
+import { Tweet } from "react-twitter-widgets"
+import _ from "lodash"
+import { useNFTNameGenerator } from "../../providers/NFTNameGeneratorProvider"
+import SkeletonCard from "../SkeletonCard"
+
+const Results = () => {
+  const { twitterId, imageURI, name, tokenId, haveTokenId } = useNFTNameGenerator()
+  const [loading, setLoading] = useState(true)
+  const openSeaLink = `https://${
+    process.env.NEXT_PUBLIC_NETWORK === "mumbai" && "testnets."
+  }opensea.io/assets/${process.env.NEXT_PUBLIC_NETWORK}/${
+    process.env.NEXT_PUBLIC_CONTRACT_ADDRESS
+  }/${tokenId}`
```

Comment:
creating open sea link to make the image card clickable. If there is a better way to do this let me know.

Reply:
this looks great 👍